core/pom.xml

```diff
@@ -370,5 +370,28 @@
             </activation>
             <!-- not including license-maven-plugin is sufficent to expose default license -->
         </profile>
+        <profile>
+            <id>release</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>attach-javadocs</id>
+                                <goals>
+                                    <goal>jar</goal>
+                                </goals>
+                                <configuration>
+                                    <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
I think we should backport the fixes that I made to get doclint fully passing on master. I found that, although we run doclint in javac, it appears it misses some issues which javadoc catches. I can backport the fixes from the PR I referenced tomorrow, or you can take a stab at backporting or merging them into this PR.